plugins/modules/network_settings_intent.py

```diff
@@ -1635,11 +1687,8 @@ def update_global_pool(self, config):
             return
 
         # Pool exists, check update is required
-        obj_params = [
-            ("settings", "settings"),
-        ]
         if not self.requires_update(self.have.get("globalPool").get("details"),
-                                    self.want.get("wantGlobal"), obj_params):
+                                    self.want.get("wantGlobal"), self.global_pool_obj_params):
```

Comment:
Where are we setting  global_pool_obj_params?.. I think this change may lead into another problem when the pool is created newly. we may not set it